manoseimas/mps_v2/views/json.py

```diff
@@ -140,6 +146,26 @@ def suggesters_json(request):
                          'subtab_counts': subtab_counts()})
 
 
+def _mangle(text):
+    text = text.lstrip()
+    text = re.sub(u'^(PAGRINDINIO )?(KOMITETO )?(IŠVADA )?', u'', text)
+    text = text.lstrip()
+    return text
+
+def _resolution_dict(resolution):
+    return {
+        'title': _mangle(resolution.title),
+        'url': resolution.source,
+    }
+
+def resolutions_json(request, suggester_slug):
+    """Get resolutions for a given suggester."""
+    resolutions_qs = CommitteeResolution.objects.filter(
+        suggestion__submitter__slug=suggester_slug).distinct()
```

Comment:
A good question.

Without `distinct` I would get multiple repetitions of resolution. I was wondering whether this has to do with the way django does lookups. Namely, perhaps it construct a natural join on the 3 tables and returns one row for each suggestion which means resolutions get duplicated.